weathersimulator-python/test_func.py

```diff
@@ -0,0 +1,13 @@
+import unittest
```

Comment:
The tests are not passing. I think that the reason is because `func.py` is a regular Flask application and here in the tests we are treating the app like a serverless function.